src/armory/Armory.js

```diff
@@ -28,83 +30,91 @@ class Armory extends React.Component<{||}> {
-                    				<option value="heavy">Heavy</option>
-                			</select>
-					<h6>Weapons</h6>
-					<select className="tankComponentMenu">
-                    				<option defaultValue></option>
-                    				<option value="1">Laser</option>
-                    				<option value="2">Machine Gun</option>
-                    				<option value="3">Vulcan Cannon</option>
-                			</select>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Laser</option>
-						<option value="2">Machine Gun</option>
-						<option value="3">Vulcan Cannon</option>
-					</select>
-					<h6>Scanner</h6>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Short Range Scanner</option>
-						<option value="2">Medium Range Scanner</option>
-						<option value="3">Long Range Scanner</option>
-					</select>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Anti-Jammer Scanner</option>
-						<option value="2">Item Scanner</option>
-					</select>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Anti-Jammer Scanner</option>
-						<option value="2">Item Scanner</option>
-					</select>
-					<h6>Jammer</h6>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Short Range Jammer</option>
-						<option value="2">Medium Range Jammer</option>
-						<option value="3">Long Range Jammer</option>
-					</select>
-					<h6>Treads</h6>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Advanced Treads</option>
-						<option value="2">Armored Treads</option>
-						<option value="3">Fast Treads</option>
-					</select>
-					<h6>Single-Use Items</h6>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Missile Tracking Beacon</option>
-						<option value="2">C4</option>
-						<option value="3">Mine</option>
-					</select>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Missile Tracking Beacon</option>
-						<option value="2">C4</option>
-						<option value="3">Mine</option>
-					</select>
-					<select className="tankComponentMenu">
-						<option defaultValue></option>
-						<option value="1">Missile Tracking Beacon</option>
-						<option value="2">C4</option>
-						<option value="3">Mine</option>
-					</select>
+
 				</div>
 			</div>
 		);
 	}
 }
 
+// Needs to be placed into armoryright
+// <ComponentMenu tank={ ["moddableLight", "machineGun", "machineGun", "shortRangeScanner", "shortRangeJammer", "mine", "c4"]} components={ ["heavy", "laser", "deathRay", "itemScanner", "longRangeJammer"] }/>
+
 export default Armory;
+
+/* Needs to be transferred to ComponentMenu.js

```

Comment:
Is there a reason for this very large amount of dead code? If not, I think we should get rid of it